src/stars.css

```diff
@@ -0,0 +1,21 @@
+[class^="stars"] {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 ;
+  margin-right: 4px;
+  background: url(./images/star.svg);
+  background-position: center !important;
+  background-repeat: no-repeat !important;
```

Comment:
it's really a bad idea to use !important. I guess your code will be work without them)